autoload/lsp/ui/vim/completion.vim

```diff
@@ -131,6 +131,7 @@ function! s:on_complete_done_after() abort
       endif
     else
       call lsp#utils#text_edit#apply_text_edits('%', [{ 'range': l:range, 'newText': l:text }])
+      call cursor(lsp#utils#position#lsp_to_vim('%', l:range.end))
     endif
```

Comment:
The snippet will place the cursor for to end of snippet (if it has no placeholders).
So we should place cursor to the range end if applying textEdit.

https://github.com/microsoft/vscode/blob/master/src/vs/editor/contrib/suggest/suggestController.ts#L374